dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/alert/AlertClientService.java

```diff
@@ -28,7 +28,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class AlertClientService {
+public class AlertClientService implements AutoCloseable {
```

Comment:
does it add component annotation?

Reply:
> does it add component annotation?

This component depends on something that are not inside this module, so we have to initialize it in the module where needed, check `dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/config/BeanConfig.java`